assets/css/media.css

```diff
@@ -1,3 +1,19 @@
+.dropped-files-info{
+    font-size: var(--size-small);
+    color: var(--button-color);
+    background-color: var(--button-background-color);
+    margin: 1px 0 0 0;
+    padding: var(--padding);
+    width: 100%;
+    border-radius: 0 0 var(--radius) var(--radius);
+}
+.dropped-files-info span:has(+ span)::after { 
+    content: ", ";
+}
+#file {width: 100%;}
+#file:has(+.dropped-files-info){
+    border-radius: var(--radius) var(--radius) 0 0;
+}
```

Comment:
For example, I would have keep all this for another PR since this is enhancement.

Reply:
I understand what you mean.
This CSS snippet should have been included in  the PR relative to the drag and drop feature.
But while cleaning the File dropdown in media, it would have been weird to me to leave unstyled the drag-n-drop info panel.

Reply:
I removed the commit from the PR

Reply:
> it would have been weird to me to leave unstyled the drag-n-drop info panel.

😁 haha yes but, from how I understand it, that's exactly the point of the commit narration: One commit should tell only one story.